modules/ROOT/pages/manage-apis-with-service-mesh.adoc

```diff
@@ -30,8 +30,11 @@ image::select-policies-for-service-mesh.png[Applying Policies to APIs]
 . Select any of the following policies to secure your APIs: 
 * *Client ID Enforcement* 
 * *JWT Validation* 
-* *Rate Limit* 
-* *Rate Limit SLA-Based* 
+* *Rate Limiting* 
+* *Rate Limiting SLA-Based* 
+* *Basic Authentication: Simple*
+* *Basic Authentication: LDAP*
+* *OpenID Connect OAuth 2.0 Token Enforcement*
```

Comment:
The only thing to point out is that to have `OpenID Connect OAuth 2.0 Token Enforcement` they need to have configured OpenID as their client management provider. Maybe we should add a note about that.

Reply:
Done! Thanks!